docs/quickstart.md

```diff
@@ -13,6 +13,7 @@ NOTE: The examples assume that you have run the setup script and that you have c
     - [Change Access Token Lifetime](#change-access-token-lifetime)
     - [Enable Account Activation via Email](#enable-account-activation-via-email)
     - [Enable Two-Factor Authentication](#enable-two-factor-authentication)
+    - [Responding to Magic Link Logins](#responding-to-magic-link-logins)
```

Comment:
How do you update the TOC?
We need tool and/or documentation for contributors.

Reply:
I ask you to pay attention to #21 again.
The fact that we do not know about _vuepress static site generater_ is not a good reason for not paying attention to the good performance of #21 
Also, we have already had a request in #373.

Reply:
I just asked how to update the TOC in the markdown file.

I don't need HTML documentation personally.
But if someone wants it, I'm fine.

Reply:
@kenjis I just use the Markdown All-in-One plugin for VS Code.